content/en/technical-advisory-group/architectural-review-committee/ARC-001.en.md

```diff
@@ -7,4 +7,163 @@ draft: false
+**Objective:** Review the Aurora initiative and secure approval for its **strategic direction and technical architecture**.
+
+---
+
+## 1. Executive Summary
+
+### Context
+
+The Aurora initiative is a universal platform for modern application hosting designed to standardize managed Kubernetes cloud offerings across the Government of Canada.
+
+Aurora is a secure, self-service (application) hosting platform that enables solution builders to rapidly develop and deploy cloud-native solutions.
+
+### Why Now?
+
+Aurora emerges at a pivotal moment, as SSC centralizes cloud teams under the new **GC Cloud One** initiative—providing the structure needed to unify fragmented efforts and scale a shared approach to cloud service delivery across government.
+
+- Departmental cloud strategies remain fragmented, **increasing cognitive load**, duplication of effort, and overall costs
+- A consistent **governance framework** is needed to reduce inefficiencies, promote **multi-cloud standardization**, and align with SSC’s **GC Cloud One** strategy
+- **Security, compliance, and technical debt** create delivery bottlenecks—manual processes, inconsistent controls, and unmanaged infrastructure increase risk and slow time-to-value
+- **Kubernetes adoption** remains challenging due to the lack of a shared platform, onboarding standards, and operational guardrails
+- A shift toward **platform engineering** is gaining momentum—departments are adopting unified platforms that embed governance, security, and self-service capabilities by design
+
+---
+
+## 2. Architecture Overview
+
+### 2.1 Technical Architecture
+
+Aurora is not a Kubernetes distribution—it is a composable set of add-on components that transform any managed Kubernetes environment into a full-featured, secure, and scalable application hosting platform. Built atop existing cloud-managed services such as AKS, EKS, GKE, and OpenShift, Aurora provides the foundational services, governance controls, and developer experience enhancements required to support modern, multi-tenant workloads across the Government of Canada.
+
+Aurora includes a curated, opinionated configuration of cloud-native components and Kubernetes resources aligned to Government of Canada requirements—balancing developer self-service with strong security, compliance, and operational guardrails.
+
+- **Base Platform:** Cloud-managed Kubernetes offerings (AKS, EKS, GKE, OpenShift)
+- **Networking Layer:** Cilium with BGP (BIRD + route reflectors) and namespace-level network policies for tenant isolation
+- **Security Model:** Zero Trust architecture, Gatekeeper (OPA) for policy enforcement, SCED integration for on-prem connectivity
+- **CI/CD Pipelines:** GitLab Runners integrated with Argo CD for GitOps-based delivery
+- **Observability Stack:** Prometheus, Loki, and centralized logging/metrics forwarding
+- **Multi-Tenancy Approach:** Namespaced resource boundaries, enforced network isolation, and RBAC + policy guardrails
```

Comment:
It might be worth mentioning that in API isolation will be looked at down the road, to provide a mechanism for those wishing to build or operate advanced services can do so without impacting other workloads (e.g., using CRDs)